spec/search-api.yaml

```diff
@@ -141,10 +152,65 @@ components:
             type: object
         pagination:
           $ref: "#/components/schemas/Pagination"
+        errors:
+          type: array
+          description: List of errors encountered
+          items:
+            $ref: "#/components/schemas/Error"
       description: A paginated collection of tabular data
-    Pagination:
+    Error:
```

Comment:
I wonder if we can align with an error representation used in other standards, e.g. https://github.com/ga4gh-discovery/ga4gh-service-registry/blob/develop/service-registry.yaml#L176-L200. It seems this could be just that + source? WDYT?

Reply:
In other places in the API using that ERROR object makes sense, but here we are essentially returning one or more errors based on what is reported by one or more connected source databses. So in this instance Status codes dont make alot of sense, since the actual response could be `200` but an error object is there with information on how to do authorization for more data